spec/support/acceptance/examples.rb

```diff
@@ -56,7 +56,11 @@
 
   describe curl_command("https://#{host_inventory['fqdn']}/pulp/api/v3/", cacert: "#{certdir}/ca-cert.pem") do
     # Requires authentication: https://github.com/pulp/pulpcore/issues/2340
-    its(:response_code) { is_expected.to eq(403) }
+    if(ENV['BEAKER_FACTER_PULPCORE_VERSION'] !~ /3.1+/)
```

Comment:
The `+` is kind of redundant since it means match `3.1` but also `3.1111`. It's a partial match so it ends up working. A stricter match is:
```suggestion
    if(ENV['BEAKER_FACTER_PULPCORE_VERSION'] !~ /^3.1[6-9]$/)
```

Reply:
Or you use the good old `['3.16', '3.17', '3.18', '3.19'].include?(…)` ;)

(I don't mind either way, just don't like regular expressions too much 🙃)

Reply:
> The `+` is kind of redundant since it means match `3.1` but also `3.1111`. It's a partial match so it ends up working. A stricter match is:

I love how this is a bug someone would have to find when pulpcore 3.100 is out.😄 
I think I like ['3.16', '3.17', '3.18', '3.19'].include?(…). Keeps it simple. Will update.

Reply:
The regex also would match `3116` ;)